### PR TITLE
Announcing 1.9.7

### PR DESCRIPTION
**Pull request by jacob-delgado**
Release notes for 1.9.7.

This will be released on Thursday July 23, 2021 at ~12pm PST

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.9.x/announcing-1.9.7/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** validator for empty regex match. ([Issue 34065](https://github.com/istio/istio/issues/34065))
+
+- **Fixed** endpointslice races leading to error state. ([Issue 33672](https://github.com/istio/istio/issues/33672))
+
+- **Fixed** endpointslice creating duplicate IPs on service update.
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.7/index.md:22]:
```suggestion
- **Fixed** `EndpointSlice` races leading to error state. ([Issue 33672](https://github.com/istio/istio/issues/33672))

- **Fixed** `EndpointSlice` creating duplicate IPs on service update.
```

File: content/en/news/releases/1.9.x/announcing-1.9.7/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - /news/announcing-1.9.7
+---
+
+This release note describes what’s different between Istio 1.9.1 and Istio 1.9.2.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** validator for empty regex match. ([Issue 34065](https://github.com/istio/istio/issues/34065))
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.7/index.md:18]: Needs "validator" added to .spelling.

**Review by craigbox — CHANGES_REQUESTED** [review]

**Review by craigbox — APPROVED** [review]

**Comment by craigbox** [comment]
LGTM, one nit: you set date to 22 July and you say you're going to release on the 23rd.

File: content/en/news/releases/1.9.x/announcing-1.9.7/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+linktitle: 1.9.7
+subtitle: Patch Release
+description: Istio 1.9.7 patch release.
+publishdate: 2021-07-22
+release: 1.9.7
+aliases:
+    - /news/announcing-1.9.7
+---
+
+This release note describes what’s different between Istio 1.9.1 and Istio 1.9.2.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.7/index.md:12]: This should be 1.9.6 and 1.9.7

**Review by ericvn — CHANGES_REQUESTED** [review]

File: content/en/news/releases/1.9.x/announcing-1.9.7/index.md (hunk 3 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.9.x/announcing-1.9.7/index.md]: Fixed

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
> you set date to 22 July

Since it also said Thursday, I suspect they really want it to be the 22nd.

**Pull request merged**